Library/Homebrew/cmd/info.rb

```diff
@@ -191,7 +191,7 @@ def json_version(version)
     version_hash[version]
   end
 
-  sig { params(all: T::Boolean, args: CLI::Args).void }
+  sig { params(all: T::Boolean, args: T.untyped).void }
```

Comment:
See https://github.com/Homebrew/brew/pull/14880/files#diff-c1211b23201bc9368c09b1da6c3fb4bb16e75fd34ad1e8500d023d753f0f544eR299 for the need to make this `untyped`